modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
OK, well I don't mind turning my `trait.mcmc` into an `mcmc.list`. And I guess I don't really even care if I have to name the column 'beta.o'. That just seems work-around-y considering the function actually works as long as `dat` ends up being a vector of posterior trait samples.